source/Core/Intermediate model/FBClassParcer.cs

```diff
@@ -98,6 +98,17 @@ private FB2SMV.FBXML.FBType Deserialie(string filename)
                 StreamReader sr = new StreamReader(filename);
                 FB2SMV.FBXML.FBType fb = (FB2SMV.FBXML.FBType) ser.Deserialize(sr);
                 sr.Close();
+                var itemToRemove = fb.InterfaceList.EventInputs.FindIndex(r => r.Name == "NDT");
+                if (itemToRemove != -1)
+                {
+                    fb.InterfaceList.EventInputs.RemoveAt(itemToRemove);
+                    VarDeclaration ndt = new VarDeclaration();
+                    ndt.Name = "NDT";
+                    ndt.Type = "NDT";
+                    ndt.ArraySize = 0;
+                    fb.InterfaceList.NonDeterministicVariable = ndt;
+                } 
+                    
```

Comment:
From the interface list event inputs, we remove "NDT" event and added NDT as NonDeterministicVariable.